src/maggma/stores/mongolike.py

```diff
@@ -564,7 +567,7 @@ def connect(self, force_reset: bool = False):
         Connect to the source data
         """
         if self._coll is None or force_reset:  # pragma: no cover
-            conn = MongoClient(self.uri)
+            conn = MongoClient(self.uri, self.mongodb_client_kwargs)
```

Comment:
I think this needs `**self.mongoclient_kwargs`, no?

Reply:
oops! fixed